sdks/java/io/snowflake/src/main/java/org/apache/beam/sdk/io/snowflake/credentials/OAuthTokenSnowflakeCredentials.java

```diff
@@ -28,4 +30,9 @@ public OAuthTokenSnowflakeCredentials(String token) {
   public String getToken() {
     return token;
   }
+
+  @Override
```

Comment:
I suggest moving changes to credentials to a separate PR since this seems to be unrelated to x-lang changes.

Reply:
I removed this improvement from this PR and I will prepare separate PR.